docs/kb/error-codes-messages.md

```diff
@@ -1627,6 +1675,150 @@ requests.
 
 **Solution**: Reset your password at https://pi.pardot.com/user/passwordReset
 
+## [](#error-code-132)Error Code: 132
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="132">Prospect is archived</err>
+</rsp>
+```
+
+**Problem**: The prospect cannot be read because it is in the recycle bin.
```

Comment:
Recommend changing the order to "The prospect is in the recycle bin and can't be read."